src/main/java/com/mraof/minestuck/item/MSFoods.java

```diff
@@ -20,6 +20,7 @@
     public static final Food FRENCH_FRY = (new Food.Builder()).hunger(1).saturation(0.1F).build();
     public static final Food STRAWBERRY_CHUNK = (new Food.Builder()).hunger(4).saturation(0.5F).build();
     public static final Food IRRADIATED_STEAK = (new Food.Builder()).hunger(4).saturation(0.4F).effect(new EffectInstance(Effects.WITHER, 100, 1), 0.9F).build();
+    public static final Food WOODEN_CARROT = (new Food.Builder()).hunger(0).saturation(0.0F).effect(new EffectInstance(Effects.INSTANT_DAMAGE, 1, 0), 1.0F).effect(new EffectInstance(Effects.INSTANT_DAMAGE, 1, 255), 0.01F).setAlwaysEdible().build();
```

Comment:
255 instant damage is way too much. Consider something else instead.

Reply:
Spontaneous idea: how about also increasing the saturation by a small amount? (0.1f or 0.2f)